src/test/java/net/kaczmarzyk/e2e/converter/CalendarE2eTest.java

```diff
@@ -101,4 +191,221 @@ public void findsByEqualDayUsingCustomConfigIgnoringTime() throws Exception {
                 .andExpect(jsonPath("$[2].firstName").value("Vanessa"))
                 .andExpect(jsonPath("$[3]").doesNotExist());
     }
+
+    @Test
+    public void findsByEqualUsingDefaultConfig() throws Exception {
+        customer("Adam", "Flayman")
+                .registrationDate(2015, 3, 4, 0, 0, 0, 0)
+                .build(em);
+
+        mockMvc.perform(get("/customers")
+                        .param("registeredEqual", "2015-03-04")
+                        .accept(MediaType.APPLICATION_JSON))
+                .andExpect(status().isOk())
+                .andExpect(jsonPath("$").isArray())
+                .andExpect(jsonPath("$[0].firstName").value("Adam"))
+                .andExpect(jsonPath("$[1]").doesNotExist());
+    }
+
+    @Test
+    public void findsByEqualWithCustomConfigContainingTime() throws Exception {
+        customer("Adam", "Flayman")
+                .registrationDate(2014, 3, 15, 23, 59, 59, 999)
+                .build(em);
+        customer("Vanessa", "Bloom")
+                .registrationDate(2014, 3, 15, 12, 34, 19, 0)
+                .build(em);
```

Comment:
I would extract saving test customers to `@Before` method so we are sure that during execution of all tests they are available. In current implementation not all tests use them (but they should). For example in `findsByBetween` you use default config for specifying date range - that's correct but in db we have only records with `registrationDate` with midnight time - we do not know if record with time part different than midnight will be returned. Maybe the part of tests name `CustomConfigContainingTime` was confusing but it refers only to format in which the date will be provided. Alternative way is to add customers with time part to other tests.